core/src/main/scala/org/bitcoins/core/wallet/builder/RawTxFinalizer.scala

```diff
@@ -401,6 +400,24 @@ case class AddFutureFeeFinalizer(
   }
 }
 
+case class SubtractFromOutputFinalizer(spk: ScriptPubKey, subAmt: CurrencyUnit)
```

Comment:
With this can we remove `SubtractFeeFromOutputsFinalizer`

Reply:
Technically yes but I actually think it could be useful in the future so I'm not planning to